src/components/typography/text/text.js

```diff
@@ -43,6 +45,7 @@ const Subheadline = props => {
         [styles.truncate]: props.truncate,
       })}
       title={props.title}
+      data-test={props.dataTest}
```

Comment:
can you use [filter-data-attributes](https://github.com/commercetools/ui-kit/blob/master/src/utils/filter-data-attributes.js) instead so that it supports all data attributes? You can just spread it like 
```
        {...filterDataAttributes(props)}
```